aws/aws.go

```diff
@@ -1,8 +1,9 @@
 package aws
 
 type AWS struct {
-	KMSKeyArn string `json:"kmsKeyArn" yaml:"kmsKeyArn"`
-	Masters   []Node `json:"masters" yaml:"masters"`
-	Region    string `json:"region" yaml:"region"`
-	Workers   []Node `json:"workers" yaml:"workers"`
+	KMSKeyArn  string   `json:"kmsKeyArn" yaml:"kmsKeyArn"`
+	Masters    []Node   `json:"masters" yaml:"masters"`
+	Region     string   `json:"region" yaml:"region"`
+	Workers    []Node   `json:"workers" yaml:"workers"`
+	PublicKeys []string `json:"publicKeys" yaml:"publicKeys"`
```

Comment:
The keys will come from the yaml file consumed by aws-operator. https://github.com/giantswarm/aws-operator/blob/master/examples/cluster.yml

The thing is that you can create and inject the keypair in AWS only when you are requesting a new instance - you cannot do this later. I would like to **not** use cloudconfig for injecting keys, since cloudconfigs are stored in bindata inside k8scloudconfig - we would need also inject them there somehow, probably also by the third-party object.